src/WebJobs.Script/Host/BlobLeaseManager.cs

```diff
@@ -153,6 +155,8 @@ private void ProcessLeaseTimerTick(object state)
             }
         }
 
+        internal static string GetBlobName(string hostId) => $"{hostId}/{LockBlobName}";
```

Comment:
can you put this under locks/{hostId}/{LockBlobName}? All the other WebJobs lease blobs are under /locks